Document steps to archive channel and save memories

SFriendLee opened the pull request.

SFriendLee committed:
Document steps to archive channel and save memories

Added steps for archiving a channel and saving memories.

Sampfluger88 reviewed (changes requested):
Couple of things missing

lukeheath commented:
@SFriendLee @Sampfluger88 Reminder to put this in draft or get it merged. Thanks!

Sampfluger88 commented on handbook/ceo/README.md:
```diff
@@ -234,6 +234,16 @@ Mike and the Fleet team
+### Archive channel and save memories
+
+Follow these steps to archive a channel and save memories (e.g. pictures or videos)
+1. Download all pictures and videos in the channel to your device you want to archive.
```
What are the exact steps for this?

Sampfluger88 commented on handbook/ceo/README.md:
```diff
@@ -234,6 +234,16 @@ Mike and the Fleet team
 5. Using the [🌧️🦉 Investors + advisors](https://docs.google.com/spreadsheets/d/15knBE2-PrQ1Ad-QcIk0mxCN-xFsATKK9hcifqrm0qFQ/edit#gid=1068113636) spreadsheet, bcc the correct individuals and send the email.
 
 
+### Archive channel and save memories
```
```suggestion
### Manually archive a Slack channel
```

SFriendLee committed:
Update handbook/ceo/README.md

Co-authored-by: Sam Pfluger <108141731+Sampfluger88@users.noreply.github.com>

SFriendLee committed:
Update handbook/ceo/README.md

Co-authored-by: Sam Pfluger <108141731+Sampfluger88@users.noreply.github.com>

SFriendLee committed:
Enhance Slack channel archiving instructions

Updated instructions for manually archiving a Slack channel with more detailed steps and a new folder naming template.

SFriendLee committed:
Revise Slack channel archiving instructions

Updated the steps for manually archiving a Slack channel, including new instructions for downloading artifacts and uploading them to Google Drive.

Sampfluger88 committed:
Apply suggestions from code review

Sampfluger88 commented:
@SFriendLee I would copy these changes to a new PR before commiting
